### PR TITLE
Only small changes in backend to add user role to jwt claims

### PR DESCRIPTION
**Pull request by not-enough-ram**

### CODE REVIEW

File: frontend/src/Pages/HolidaysPage.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components/macro";
+import HolidayList from "../components/HolidayList";
+
+export default function HolidaysPage() {
+    return (
+        <Wrapper>
+            <HolidayList/>
```
Comment (Cathesso) [inline frontend/src/Pages/HolidaysPage.js:7]: What is a "Holiday"?

**Review by Cathesso — APPROVED** [review]
Looks good to me

**Review by Cathesso — APPROVED** [review]

File: backend/src/main/java/de/neuefische/backend/controller/LoginController.java [hunk 2]
```diff
@@ -14,27 +14,32 @@
...
 
     @PostMapping("login")
     public String login(@RequestBody AppUser data) {
         try {
             UsernamePasswordAuthenticationToken authentication = new UsernamePasswordAuthenticationToken(data.getUsername(), data.getPassword());
             authenticationManager.authenticate(authentication);
-            String token = jwtService.createToken(new HashMap<>(), data.getUsername());
+            AppUser appUser = appUserRepository.findById(data.getUsername()).get();
+            String userRole = appUser.getUserRole().toString();
+            String token = jwtService.createToken(new HashMap<>(Map.of("userRole", userRole)), data.getUsername());
```
Comment (fabianschmauder) [inline backend/src/main/java/de/neuefische/backend/controller/LoginController.java:42]: i think role would be enough

File: frontend/package.json [hunk 3]
```diff
@@ -38,6 +40,8 @@
     ]
   },
   "devDependencies": {
+    "axios": "^0.21.1",
+    "prettier": "2.3.1",
     "react-router-dom": "^5.2.0"
```
Comment (fabianschmauder) [inline frontend/package.json:45]: you need axios and routing only in dev? 😉

**Review by fabianschmauder — COMMENTED** [review]
LGTM ☀️😎

**Pull request merged**